test/test-remote/looker/src/metrics/dummyseries.ts

```diff
@@ -418,11 +418,11 @@ export class MetricSeries extends TimeseriesBase {
       fragment.addSample(this.nextSample());
     }
 
-    const genduration = mtimeDiffSeconds(t0);
-    log.debug(
-      "MetricSeriesFragment addSample loop took: %s s",
-      genduration.toFixed(3)
-    );
+    //const genduration = mtimeDiffSeconds(t0);
+    //log.debug(
+    //  "MetricSeriesFragment addSample loop took: %s s",
+    //  genduration.toFixed(3)
+    //);
```

Comment:
I might reactivate this : ) it's a simple benchmark for some core pieces -- of course this yields larger numbers only for extreme cases, which I had done for testing locally.